src/toil/jobStores/aws/jobStore.py

```diff
@@ -299,6 +339,73 @@ def deleteFile(self, jobStoreFileID):
         else:
             info.delete()
 
+    nativeUrlRegex = re.compile(r"""
+        # url scheme
+        [sS]3://
+
+        # bucket name
+        #   (1) Must have length 3-255
```

Comment:
"The rules for bucket names in the US East (N. Virginia) region allow bucket names to be as long as 255" Just covering all the bases